pkg/util/utils_supported.go

```diff
@@ -60,6 +60,36 @@ func GetRootlessRuntimeDir() (string, error) {
 	return rootlessRuntimeDir, nil
 }
 
+// GetRootlessConfigHomeDir returns the config home directory when running as non root
+func GetRootlessConfigHomeDir() (string, error) {
+	var rootlessConfigHomeDirError error
```

Comment:
`rootlessConfigHomeDirError` isn't global and the code below only runs once, so if it returns an error, subsequent calls will always return `"", nil`.

Reply:
I think that's OK though.  @giuseppe I stole this methodology from your GetRootlessRuntimeDir() function, should I not be doing the same here or does that need to be adjusted too?